custom/icds_reports/sqldata/exports/dashboard_usage.py

```diff
@@ -189,7 +189,7 @@ def get_excel_data(self):
 
             usernames = [user['username'] for user in users if dashboard_uname_rx.match(user['username'])]
 
-            records = list(ICDSAuditEntryRecord.objects.filter(url='/a/icds-dashboard-qa/cas_export',
+            records = list(ICDSAuditEntryRecord.objects.filter(url='/a/' + self.domain + '/cas_export',
```

Comment:
this looks good. just two comments.
1) using format could be more readable
2) can you change `cas_export` to `icds_export_indicator`. the Url was changed after you started building the report.
@challabeehyv

Reply:
@Rohit25negi  should i harcode the 'icds_export_indicator' value or is there a way i can get it in dynamic?

Reply:
you can hardcode its a url

Reply:
done.please check